.github/workflows/test.yml

```diff
@@ -163,14 +163,19 @@ jobs:
     - name: Install deps
       # Not using setup-msys2 `install` option to make it easier to copy/paste
       run: |
-        pacman --noconfirm -S --needed git rust
+        pacman --noconfirm -S --needed git rust python3 diffutils tmux
```

Comment:
looks good to me overall.

I'll probably use actions-tmate to ssh into the runners to debug any future failures